networks.json

```
@@ -43,7 +43,7 @@
     "1": {
       "events": {},
       "links": {},
-      "address": "0x60fBbd1FB0076971E8060631b5dD895F55aD5ab7",
+      "address": "0x60fbbd1fb0076971e8060631b5dd895f55ad5ab7",
```

Why did it suddenly become not checksum?

---

Yea, perhaps we can just revert this change...
```suggestion
      "address": "0x60fBbd1FB0076971E8060631b5dD895F55aD5ab7",
```